resources/policy/bundles/asm-policy-v0.0.1/README.md

```diff
@@ -0,0 +1,31 @@
+# Anthos Service Mesh Policy v0.0.1
```

Comment:
Let's clear this readme and write a fresh one (doesn't have to be in this PR though)

Reply:
done